cmd/commands/dev.go

```diff
@@ -88,10 +89,14 @@ func doDev(cmd *cobra.Command, args []string) {
 	tick := viper.GetInt("tick")
 
 	if err := itrace.NewUserTracer(ctx, itrace.TracerOpts{
-		ServiceName:   "devserver",
-		Type:          itrace.TracerTypeOTLPHTTP,
+		ServiceName: "devserver",
+		Type:        itrace.TracerTypeOTLPHTTP,
+		// Type:          itrace.TracerTypeNATS,
 		TraceEndpoint: "localhost:8288",
 		TraceURLPath:  "/dev/traces",
+		NATS: &exporters.NatsExporterOpts{
+			Subjects: []string{"inngest.run.spans"},
+		},
```

Comment:
**question**: when this is added, will it swap out to an in-memory pubsub via the gocloud package when running the dev server? Maybe I'm just not sure how this was working before with the previous tracer anyway.

Reply:
nope it won't. to enable NATS, the `Type` a couple of lines above also need to be changed to `itrace.TracerTypeNATS`.
So just setting this configuration won't have any impact.